modules/rosa-policy-change-management.adoc

```diff
@@ -126,7 +126,7 @@ You can review the history of all cluster upgrade events in the {cluster-manager
 - Set up and configure Amazon VPC components required to provision the cluster, such as subnets, load balancers, internet gateways, and NAT gateways.
 
 - Provide the ability for the customer to
-manage AWS VPN connectivity with on-premises resources, Amazon VPC-to-VPC connectivity, and AWS Direct Connect as required through  {cluster-manager}.
+manage AWS VPN connectivity with on-premise resources, Amazon VPC-to-VPC connectivity, and AWS Direct Connect as required through  {cluster-manager}.
```

Comment:
```suggestion
manage AWS VPN connectivity with on-premises resources, Amazon VPC-to-VPC connectivity, and AWS Direct Connect as required through  {cluster-manager}.
```

That should remove this. Otherwise try `rm modules/rosa-policy-change-management.adoc; git checkout upstream/main -- modules/rosa-policy-change-management.adoc`